core/api/src/services/mongoose/schema.types.d.ts

```diff
@@ -63,6 +63,7 @@ interface WalletInvoiceRecord {
   selfGenerated: boolean
   pubkey: string
   paid: boolean
+  paymentRequest?: string
```

Comment:
```suggestion
  paymentRequest?: string // optional because we didn't stored it in the past
```

maybe worth adding a comment on why we're making it optional?